objects.go

```diff
@@ -579,3 +579,78 @@ func (element *Element) parse(line string) error {
 	}
 	return nil
 }
+
+// Object implementation for Flowtable
+func (flowtable *Flowtable) validate(verb verb) error {
+	switch verb {
+	case addVerb, createVerb:
+		if flowtable.Name == "" {
+			return fmt.Errorf("no name specified for flowtable")
+		}
+		if flowtable.Handle != nil {
+			return fmt.Errorf("cannot specify Handle in %s operation", verb)
+		}
+	case deleteVerb:
+		if flowtable.Name == "" && flowtable.Handle == nil {
+			return fmt.Errorf("must specify either name or handle")
+		}
+	default:
+		return fmt.Errorf("%s is not implemented for flowtables", verb)
+	}
+
+	return nil
+}
+
+func (flowtable *Flowtable) writeOperation(verb verb, ctx *nftContext, writer io.Writer) {
+	// Special case for delete-by-handle
+	if verb == deleteVerb && flowtable.Handle != nil {
+		fmt.Fprintf(writer, "delete flowtable %s %s handle %d", ctx.family, ctx.table, *flowtable.Handle)
+		return
+	}
+
+	fmt.Fprintf(writer, "%s flowtable %s %s %s", verb, ctx.family, ctx.table, flowtable.Name)
+	if verb == addVerb || verb == createVerb {
+		fmt.Fprintf(writer, " {")
+
+		if flowtable.Priority != nil {
+			// Parse the priority to a number if we can, because older
+			// versions of nft don't accept certain named priorities
+			// in all contexts (eg, "dstnat" priority in the "output"
+			// hook).
+			if priority, err := ParsePriority(ctx.family, string(*flowtable.Priority)); err == nil {
```

Comment:
`ParsePriority` is designed around the idea that there are two sets of priorities that we care about, the ones for `BridgeFamily` and the ones for everyone else. But `add flowtable` has its own separate list (which only contains `filter`). Maybe add an `objType` arg to `ParsePriority`?

OTOH, the comment isn't really relevant here, since there's only a single symbolic value allowed and all versions of nft understand it... so we could just not bother trying to parse here, and just use the provided priority literally.

Reply:
ok